.github/workflows/build-test.yml

```diff
@@ -104,7 +107,7 @@ jobs:
           sbt test
 
   test-yugabyte:
-    name: Run tests with Yugabyte
+    name: Run tests with Yugabyte on JDK 11
```

Comment:
Remember to update the branch protection rules to set these as required. For that reason I'm not convinced that it's worth having exact names on these, but not a strong opinion.